### PR TITLE
Azure governance docs

### PR DESCRIPTION
**Pull request by joeyouss**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [X] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

**Comment by iamanmolm** [comment]
I think this was the not the expected structure. The structure was expected like what we have specs. @Rohit-Kaliki ?

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md [hunk 1]
```diff
@@ -3,192 +3,336 @@ title: Overview
...
 
-At a high level, Cloud Custodian integrates seamlessly with services, features, and resources native to AWS, Azure, and GCP. The platform's basic syntax of resources, filters, and actions eliminates the complexity of API calls, business logic, and data translation, making cloud policy authorship more straightforward and efficient.
+Harness also offers a wide range of policies which are available out of the box, which you can leverage on day 0 to optimize your cloud resources and set up guardrails against future wastage.
 
+#### AWS Resource Coverage (Comprehensive list)
 
-:::info important
-Currently, Harness supports AWS, Azure and GCP cloud governance.
-:::
+EC2 instances
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:21]: We should make this as a list.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md [hunk 2]
```diff
@@ -3,192 +3,336 @@ title: Overview
...
+S3 buckets
+Lambda functions
+RDS (Relational Database Service) instances
+CloudFormation stacks
 
-## Anatomy of a cloud asset governance rule
+#### Azure Resource Coverage (Comprehensive list)
 
-Cloud policies use a declarative vocabulary of resources, filters, and actions to be configured in YAML. This vocabulary is explained in the table below:
+Virtual Machines (VMs)
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:29]: We should make this as a list.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md [hunk 3]
```diff
@@ -3,192 +3,336 @@ title: Overview
...
-  </tr>
-  <tr>
-   <td>actions   </td>
-   <td>In Cloud Custodian, actions are operations that can be performed on a resource and are applied to the output of the filters specified in the policy. Actions can include things like terminating an EC2 instance, deleting an S3 bucket, or sending an email notification.<p>Actions in Cloud Custodian are essentially webhooks that are executed when the criteria specified in the policy are met. These webhooks can also be used more generically, allowing the automation of a wide range of tasks. </p>   </td>
-  </tr>
-</table>
+#### GCP Resource Coverage (Comprehensive list)
 
+Compute Engine instances
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:37]: We should make this as a list.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md [hunk 4]
```diff
@@ -3,192 +3,336 @@ title: Overview
...
-Cost savings are calculated for the last 30 days. However, for rightsizing rules, it depends on the resize percentage specified in the policy. For example, let's consider the following rule to resize an RDS instance. The potential or realized cost savings is 30% of the last 30 days cost.
+Cloud Custodian comes with all the goodness of battle testing by the community & detects and auto remediates issues - it does come with its own set of challenges. Let’s dive into what are the key challenges that organizations run into when leveraging Cloud Custodian at scale to manage their assets.
 
-```
-policies:
-  - name: rds-resize-down
-    resource: rds
+### Harness vs Cloud Custodian
+
+Cloud Custodian, while a widely used open-source cloud management tool, presents several challenges, including lack of a graphical interface, scalability issues, limited reporting and security features, complex policy creation requiring YAML syntax knowledge, and operational overhead.
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:72]: This comparison can be framed in a better way. Paragraphs are not that intuitive.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md [hunk 5]
```diff
@@ -3,192 +3,336 @@ title: Overview
...
+
+Additionally, Harness incorporates Out-of-the-Box Recommendations to identify cost-saving opportunities and improve compliance and security. By choosing Harness Cloud Asset Governance, organizations can optimize their cloud governance, enhance customization and usability, and overcome the challenges associated with self-hosting Cloud Custodian.
+
+Harness Cloud Asset Governance streamlines cloud management processes, improves governance efficiency, and enables organizations to achieve a well-managed cloud environment effectively. More details about the comparison can be found here.
+
+## Cost Savings
+
+### Recommendations
+
+1. **Recommendation: delete-low-utilised-cosmodb**
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:84]: Point 1 looks bigger than Point 2. Also sub point is going into same line as Point 1.
Screenshot 2024-03-08 at 9 04 19 AM

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md [hunk 6]
```diff
@@ -3,192 +3,336 @@ title: Overview
...
 
-* `delete-snapshot-with-no-volume
-`
-```
-policies:
-  - name: delete-snapshot-with-no-volume
-    description: Find any snapshots that do not have a corresponding volume.
-    resource: aws.ebs-snapshot
+
+2. **Recommendation:**
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:110]: It is missing recommendation name. Also point and sub points are coming in the same line.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md [hunk 7]
```diff
@@ -3,192 +3,336 @@ title: Overview
...
+  a. **Description**: Delete all unattached disks
+  b. **Policy Used**:
+  ```
+
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:114]: Remove extra space from here.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md [hunk 8]
```diff
@@ -3,192 +3,336 @@ title: Overview
...
     actions:
-      - delete
-```
-* `delete-snapshot-unused`
-```
-policies:
-  - name: delete-snapshot-unused
-    resource: ebs-snapshot
+      - type: delete
+
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:126]: Also remove extra space from here.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md [hunk 9]
```diff
@@ -3,192 +3,336 @@ title: Overview
...
+  ```
+  c. **Savings Computed**:The recommendation identifies a list of resources; to calculate potential savings, the costs of all resources over the last 30 days is summed together and that is shown as the potential savings.
+
+  d. **Permissions Required**: For running actions, use the Contributor Role; for running filters and generating recommendations, use the Reader Role.
+
+
+
+
+3. **Recommendation:delete-low-utilised-load-balancers**
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:135]: Point and sub points are coming in the same line.

Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:135]: Also how come sub points are going outside the points:
Screenshot 2024-03-08 at 9 07 58 AM

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md [hunk 10]
```diff
@@ -3,192 +3,336 @@ title: Overview
...
+          timeframe: 72
+      actions:
+        - type: delete      
+    ```
+  c. **Savings Computed**: The recommendation identifies a list of resources; to calculate potential savings, the costs of all resources over the last 30 days is summed together and that is shown as the potential savings.
+
+  d. **Permissions Required**: For running actions, use the Contributor Role; for running filters and generating recommendations, use the Reader Role.
+
+
+4. **Recommendation:delete-orphaned-network-interface**
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:159]: Point and sub points are coming in the same line. Policy seems to be outside the code blocks and further subheading went inside the code blocks.
Screenshot 2024-03-08 at 9 09 53 AM

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md [hunk 11]
```diff
@@ -3,192 +3,336 @@ title: Overview
...
+          value: null
+      actions:
+        - type: delete
+
+    ```
+  c. **Savings Computed**: The recommendation identifies a list of resources; to calculate potential savings, the costs of all resources over the last 30 days is summed together and that is shown as the potential savings.
+
+  d. **Permissions Required**: For running actions, use the Contributor Role; for running filters and generating recommendations, use the Reader Role.
+
+5. **Recommendation:stop-underutilized-vms**
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:180]: Everything below this is completely messed up. Points are within code blocks and policies are outside code blocks.
Screenshot 2024-03-08 at 9 12 09 AM

**Review by iamanmolm — CHANGES_REQUESTED** [review]
Kindly change the structure and also address all the comments above.

**Comment by joeyouss** [comment]
Closing this PR since azure docs will be updated over 4 PRs

**Pull request closed without merging**